requirements.txt

```diff
@@ -10,3 +11,4 @@ sentry-sdk[Flask]==1.5.1
 dbus-python==1.2.16
 hm-pyhelper==0.13.16
 python-gnupg==0.4.8
+grpcio==1.44.0
```

Comment:
This probably belongs as part of a different PR. it's not related to Docker caching.

Reply:
Yes it was there for testing that wheels work with the venv change.

Reply:
removed it.